hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -519,6 +519,11 @@ public final class OzoneConfigKeys {
   public static final String OZONE_MANAGER_FAIR_LOCK = "ozone.om.lock.fair";
   public static final boolean OZONE_MANAGER_FAIR_LOCK_DEFAULT = false;
 
+  public static final String OZONE_MANAGER_STRIPED_LOCK_SIZE_PREFIX =
+      "ozone.om.lock.stripes.count.";
```

Comment:
```suggestion
      "ozone.om.lock.stripes.count";
```

Reply:
I would prefer `ozone.om.lock.stripes` (without `.count`), see https://github.com/apache/ozone/pull/4885#discussion_r1239365508.

And this is a prefix, so `.` is needed at the end.  Full config key is derived by appending resource name.

Reply:
Thanks for the reminder @adoroszlai. I even forgot about that conversation. Updated.